arch/src/aarch64/layout.rs

```diff
@@ -106,7 +106,7 @@ pub const RSDP_POINTER: GuestAddress = GuestAddress(ACPI_START);
 pub const KERNEL_START: u64 = ACPI_START + ACPI_MAX_SIZE as u64;
 
 /// Pci high memory base
-pub const PCI_HIGH_BASE: u64 = 0x80_0000_0000_u64;
+pub const PCI_HIGH_BASE: u64 = 0x1_0000_0000_u64;
```

Comment:
@michael2012z I know we have talked about this but please confirm I am doing the correct change. Thank you :))